src/labels/CustomLabels.labels

```diff
@@ -5157,6 +5157,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>campaignMemberStatusResponded</shortDescription>
         <value>campaignMemberStatusResponded</value>
     </labels>
+    <labels>
+        <fullName>CallableApiJobScheduleInvalidClass</fullName>
+        <categories>API</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>CallableApi Job.Schedule Invalid Class</shortDescription>
+        <value>An Apex class for &quot;{0}&quot; cannot be found or does not implement the Scheduleable interface</value>
```

Comment:
@npsp-reedestockton - Just so I understand, the {0} variable here is the name of an invalid class?

Reply:
@sam-knox That's correct.  The parameter is whatever the caller passed in as the apex class name.